conductor/blueprints/package/controllers.py

```diff
@@ -32,25 +75,36 @@ def upload(datapackage, callback, token, cache_set):
             "error": 'unauthorized'
         }
     else:
-        params = {
-            'package': datapackage,
-            'callback': callback
+        key = 'os-conductor:package:'+datapackage
+        ret = {
+            "progress": 0,
+            "status": "queued"
         }
-        load_url = os_api_url+'/api/3/loader/'
```

Comment:
Does this mean we no longer need the os-api-loader service (the os-api app configured as a loader)?

Reply:
indeed. The entire service can be removed from os-api

Reply:
@brew @akariv should we perhaps merge os-api and os-conductor into a single codebase?

Reply:
With this change these services are completely disconnected - 
so it became quite simple to do it (while there's no special gain from doing so)

Reply:
Removed all loader functionality in the corresponding os-api pull request.